docs/admin/cluster-components.md

```diff
@@ -146,6 +195,9 @@ network rules on the host and performing connection forwarding.
 `monit` is a lightweight process babysitting system for keeping kubelet and docker
 running.
 
+### fluentd
+
+`fluentd` is a daemon which helps provide [Cluster-level logging](#cluster-level-logging).
```

Comment:
Maybe say "is a daemon that runs on every Kubernetes node"
(though it's a bit redundant)